test/RibbonThetaVault.js

```diff
@@ -643,15 +668,24 @@ function behavesLikeRibbonOptionsVault(params) {
       });
     });
 
+    describe("#isYearnWrapped", () => {
+      it("returns the correct flag for whether vault wraps deposits into yearn token", async function () {
+        assert.equal(await this.vault.isYearnWrapped(), this.isYearnWrapped);
+      });
+    });
+
     describe("#delay", () => {
       it("returns the delay", async function () {
         assert.equal((await this.vault.delay()).toNumber(), OPTION_DELAY);
       });
     });
 
-    describe("#asset", () => {
+    describe("#collateralAsset", () => {
       it("returns the asset", async function () {
-        assert.equal(await this.vault.asset(), this.collateralAsset);
+        assert.equal(
+          (await this.vault.collateralToken()).address,
+          this.collateralAsset
+        );
       });
     });
 
```

Comment:
lmk whatever else you think is worth testing that would be unique to yearn wrap

Reply:
Interested in testing out the different scenarios/state transitions for wrapping and unwrapping:

1. User deposits, then immediately withdraws (unwraps 0)
2. User deposits, rolled by manager, withdraws after roll (unwraps withdrawal amount)
3. User deposits, rolled by manager, withdraws entire 10% after roll (unwraps up to 10%)